pkgs/development/interpreters/python/hooks/pip-install-hook.sh

```diff
@@ -1,17 +1,27 @@
 # Setup hook for pip.
-echo "Sourcing pip-install-hook"
+# shellcheck shell=bash
 
-declare -a pipInstallFlags
+echo "Sourcing pip-install-hook"
 
 pipInstallPhase() {
     echo "Executing pipInstallPhase"
     runHook preInstall
 
+    # shellcheck disable=SC2154
```

Comment:
Not blocking, but: would be nice to do a trick like that `/dev/null` thing to get declarations of all these essential variables.